### ISSUE TITLE
Redundant import of Control.Applicative

### ISSUE THREAD

**Issue by triplepointfive** [ndmitchell/hlint#145]
Since base 4.8.0.0 `<$>` is already included in Prelude, but hlint suggests to import it anyway,

```
Warning: Use <$>
Found:
  fmap head $ runX (doc >>> getWorld)
Why not:
  head Control.Applicative.<$> runX (doc >>> getWorld)
```

**Comment by ndmitchell** [comment]
Only for people using GHC 7.10 exclusively, and most people tend to support GHC 7.8 with code they are writing. It's certainly a change worth making, the only question is if I should be making it now, or when 7.12 comes out.

**Comment by Heather** [comment]
there are few ways to avoid warning, for example

``` haskell
import Control.Applicative -- needed in 7.8, and not a warning in 7.10
                           -- because it comes before the Prelude import,
                           -- though it actually is redundant in 7.10
import Prelude             -- has the effect of canceling the implicit
                           -- import of Prelude
```

from https://ghc.haskell.org/trac/ghc/ticket/8363

**Comment by ndmitchell** [comment]
I use that myself in hlint: https://github.com/ndmitchell/hlint/blob/636b0337344e1c6609b1d86474d57936defbccde/src/Hint/List.hs#L30-L32 :)

**Comment by ndmitchell** [comment]
This got fixed some time in the distant past.
